face_detection_masking/test-face-detection-masking.py

```diff
@@ -0,0 +1,119 @@
+import unittest
+import unittest.mock
+import numpy as np
+import cv2
+
+import main
+import exceptions
+
+
+class TestFaceDetection(unittest.TestCase):
+    def test_smiling_face(self):
+        uri = "https://image.freepik.com/free-photo/smiling-man-face-white-background_33839-3342.jpg"
+        try:
+            main.face_detection(uri)
```

Comment:
You can do an assert if face_detection returns anything. Or you can find out some indicators for successful face detection.